entrypoint/entrypoint/cli.py

```diff
@@ -51,6 +51,13 @@ def init(sys_argv=None) -> argparse.Namespace:
     parser.add_argument("--timeout", type=str, default="600",
                         help="The amount of time in seconds that inspector-sbomgne will run. When this timeout is exceeded, sbomgen will gracefully conclude and present any findings discovered up to that point.")
 
+    parser.add_argument("--platform", type=str,
+                        help="Specifies the OS and CPU arch of the container image you wish to scan. Valid inputs are "
+                             "of the form 'os/cpu/variant' for example, 'linux/amd64', 'linux/arm64/v8', etc. If no platform is "
+                             "specified, the system will use the same platform as the host that is performing the "
+                             "scan. This argument only affects container image scans. Requires inspector-sbomgen "
+                             "1.5.1 or later.")
```

Comment:
[nit] as we shipped 1.5.2, should this be updated to reflect?

Reply:
inspector-sbomgen 1.5.1 is the **minimum** version number that supports `--platform`.
1.5.2 didn't make any changes to platform handling, so I think listing it is unnecessary in this case.
Thanks for reviewing.